docs/user-manual/en/masking-passwords.md

```diff
@@ -257,7 +257,26 @@ codec other than the default one. For example
 With this configuration, both passwords in ra.xml and all of its MDBs will have
 to be in masked form.
 
-### login.config
+### PropertiesLoginModule
+Artemis supports Properties login module to be configured in JAAS configuration file
+(default name is `login.config`). By default, the passwords of the users are in plain text
+or masked with the DefaultSensitiveStringCodec.
+
+Set the `org.apache.activemq.jaas.properties.password.codec` property to set a custom codec,
+i.e. to use the `org.apache.activemq.artemis.tests.integration.security.MD5SensitiveDataCodec` codec
```

Comment:
```suggestion
To use a custom codec class, set the `org.apache.activemq.jaas.properties.password.codec` property to the class name
e.g. to use the `com.example.MySensitiveDataCodecImpl` codec class:
```
Related to other comment, it definitely seems like we shouldnt be adding docs pointing to a class using MD5. Both since there seems little need using a real class (which is only part of the tests) at all, and since inevitably someone might actually go find it and just use what its doing.

Might also be worth linking to the section on custom codecs, since it wasnt covered yet and comes later.